deepchecks/checks/methodology/unused_features.py

```diff
@@ -64,8 +65,26 @@ def __init__(self, feature_importance_threshold: float = 0.2, feature_variance_t
         self.n_top_unused_to_show = n_top_unused_to_show
         self.random_state = random_state
 
-    def run_logic(self, context: CheckRunContext) -> CheckResult:
-        """Run check.
+    def run(self,
+            train_dataset: Dataset,
+            test_dataset: Dataset,
+            model: BasicModel = None,
+            feature_importance_force_permutation: bool = None,
```

Comment:
```suggestion
            feature_importance_force_permutation: bool = False,
```

Reply:
The idea of having it none is to have the default in a single place in the context init